src/test/kotlin/org/rust/ide/inspections/RsConstOrTypeParamsInExternTest.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+class RsConstOrTypeParamsInExternTest : RsInspectionsTestBase(RsConstOrTypeParamsInExternInspection::class) {
```

Comment:
Previously, we used single test class for all errors produced by an annotator. For example, `RsSyntaxErrorsAnnotatorTest` or `RsErrorAnnotatorTest`. It may lead to a huge file. So I propose not to merge these tests into `RsSyntaxErrorsAnnotatorTest` and keep it separately like it's done currently